.github/workflows/bioccheck.yaml

```diff
@@ -53,6 +53,20 @@ on:
         required: false
         type: string
         default: "."
+      deps-installation-method:
+        description: |
+          Which method for installing R package dependencies to use? Supported values are:
+          staged-dependencies
+          setup-r-dependencies
+        required: false
+        type: string
+        default: staged-dependencies
+      lookup-refs:
+        description: |
+          List of package references to be used by setup-r-dependencies action if deps-installation-method == 'setup-r-dependencies'.
```

Comment:
I would phrase it as follows:
Passed to `insightsengineering/setup-r-dependencies`. See its documentation. Used only if `deps-installation-method == 'setup-r-dependencies'`

Reply:
Okay.